apps/web/components/ServiceInformation/ServiceInformation.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import * as styles from './ServiceInformation.treat'
+import {
+  Box,
+  GridColumn,
+  GridRow,
+  Inline,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import { ApiService } from '@island.is/api/schema'
```

Comment:
This should probably be imported from the web's schema `@island.is/web/graphql/schema`.
You generate schema with codegen:
`yarn nx run api:schemas/codegen --skip-nx-cache && yarn nx run web:schemas/codegen --skip-nx-cache`